requirements.txt

```diff
@@ -2,3 +2,4 @@ boto3>=1.4.0
 cached_property
 python-keystoneclient>=1.8.1
 python-swiftclient
+futures>=3.0.5; python_version < '3.0'
```

Comment:
The benefits of futures don't seem be matched by the cost of adding another dependency.